sys/sys/elf_common.h

```diff
@@ -383,6 +383,21 @@ typedef struct {
 #define	EF_RISCV_RVE		0x00000008
 #define	EF_RISCV_TSO		0x00000010
 
+// Definitions from LoongArch ELF psABI v2.01.
+// Reference: https://github.com/loongson/LoongArch-Documentation
+// (commit hash 296de4def055c871809068e0816325a4ac04eb12)
+
+// LoongArch Base ABI Modifiers
+#define EF_LOONGARCH_ABI_SOFT_FLOAT     0x00000001
+#define EF_LOONGARCH_ABI_SINGLE_FLOAT   0x00000002
+#define EF_LOONGARCH_ABI_DOUBLE_FLOAT   0x00000003
+#define EF_LOONGARCH_ABI_MODIFIER_MASK  0x00000007
+
+// LoongArch Object file ABI versions
+#define EF_LOONGARCH_OBJABI_V0          0x00000000
+#define EF_LOONGARCH_OBJABI_V1          0x00000040
+#define EF_LOONGARCH_OBJABI_MASK        0x000000C0
```

Comment:
> This part of the change should be split out into its own commit, as elf_common.h is from FreeBSD and not elftoolchain.
> 
> BSD style prefers C-style comments, i.e., `/* Definitions from ... */` instead of `//`.

Okay, I understand.

I have one more question: 
Since we are making split commits, can I also commit other parts that might be related?
For example: relocation types.

```

 /* GNU note types. */
 #define        NT_GNU_ABI_TAG          1
@@ -1379,6 +1402,123 @@ typedef struct {
 #define        R_RISCV_32_PCREL        57
 #define        R_RISCV_IRELATIVE       58

+/* LoongArch relocation types used by the dynamic linker */
+#define R_LARCH_NONE                         0
+#define R_LARCH_32                           1
+#define R_LARCH_64                           2
+#define R_LARCH_RELATIVE                     3
+#define R_LARCH_COPY                         4
+#define R_LARCH_JUMP_SLOT                    5
+#define R_LARCH_TLS_DTPMOD32                 6
+#define R_LARCH_TLS_DTPMOD64                 7
+#define R_LARCH_TLS_DTPREL32                 8
+#define R_LARCH_TLS_DTPREL64                 9
+#define R_LARCH_TLS_TPREL32                 10
+#define R_LARCH_TLS_TPREL64                 11
+#define R_LARCH_IRELATIVE                   12
+#define R_LARCH_MARK_LA                     20
+#define R_LARCH_MARK_PCREL                  21
+#define R_LARCH_SOP_PUSH_PCREL              22
+#define R_LARCH_SOP_PUSH_ABSOLUTE           23
+#define R_LARCH_SOP_PUSH_DUP                24
+#define R_LARCH_SOP_PUSH_GPREL              25
+#define R_LARCH_SOP_PUSH_TLS_TPREL          26
```

Thank you

Reply:
Yes, I think this is ok.